frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1996,7 +2052,7 @@ mod tests {
 			roll_to(14);
 			assert_eq!(MultiPhase::current_phase(), Phase::Off);
 
-			roll_to(15);
+			roll_to_signed();
```

Comment:
The assertions above this line can go away

Reply:
There are lot of other cases like this where we don't need to assert the phase before/after `roll_to_<event/phase>`. These are already tested once in the `phase_rotation_works` and redundant in other cases. Having them implies we need to mentally keep track of block number, which is what we want to move away from in the first place.